sentry-android-core/src/main/java/io/sentry/android/core/SentryAndroidOptions.java

```diff
@@ -32,6 +33,11 @@
   /** Enable or disable automatic breadcrumbs for App Components Using ComponentCallbacks */
   private boolean enableAppComponentBreadcrumbs = true;
 
+  public SentryAndroidOptions() {
+    setSentryClientName(BuildConfig.SENTRY_CLIENT_NAME + "/" + BuildConfig.VERSION_NAME);
+    setSdkInfo(SdkInfo.createSdkInfo(BuildConfig.SENTRY_CLIENT_NAME, BuildConfig.VERSION_NAME));
```

Comment:
This should be an override to `SentryOptions` ctor which sets it to `sentry.java`